components/Footer.tsx

```diff
@@ -1,32 +1,64 @@
 import React from "react";
 import styles from "../styles/Footer.module.css";
+import Image from "next/image";
 
 export default function Footer() {
 	return (
 		<div className={styles.globalFooter}>
-			<div className={styles.medexLogo}>Medex</div>
 			<div className={styles.searchBoxLeft}>
-				<span className={styles.smallText}>
-					What are you looking for?
-				</span>
-				<input type="text" />
+				<h2>Medex</h2>
+				<p className={styles.smallText}>What are you looking for?</p>
+				<input
+					type="text"
+					name="search"
+					placeholder="Search.."
+					className={styles.inputSearch}
+				/>
```

Comment:
ten input musi być otoczony <form>-em i musi gdzieś mieć możliwość obsłużenia wyszukwiania, teraz to spełnia niby ikonka, ale nie ma jak jej użyć, czyli dumb & smart component jest zbyt dumb, żeby podłączyć go z jakimś "smart"

Reply:
solved: https://github.com/arjamizo/medex-langing-page/pull/2/commits/e53f2b8a4fe335865eaab23453bc5a31a1ba4db4 

https://user-images.githubusercontent.com/45758580/134776043-882b31d9-7367-4228-9846-6f394157bc99.mp4